frontend/src/components/teacher/student-management/classrooms/ClassroomCard.tsx

```diff
@@ -23,7 +23,7 @@ import ClassroomPopover from "./ClassroomPopover";
 interface ClassroomCardProps {
   id: string;
   name: string;
-  startDate?: string;
+  startDate?: Date;
```

Comment:
what's the context for making this change?

Reply:
its because the return from the `classesByTeacher` mutation is a `ClassResponseDTO`, where startDate is type `date`.